tests/unit/s2n_3des_test.c

```diff
@@ -58,18 +58,17 @@ int main(int argc, char **argv)
     EXPECT_SUCCESS(s2n_hmac_init(&conn->secure.server_record_mac, S2N_HMAC_SHA1, mac_key, sizeof(mac_key)));
     conn->actual_protocol_version = S2N_TLS11;
 
-    int max_aligned_fragment = S2N_DEFAULT_FRAGMENT_LENGTH - (S2N_DEFAULT_FRAGMENT_LENGTH % 8);
-    for (int i = 0; i <= max_aligned_fragment + 1; i++) {
+    for (int i = 0; i <= S2N_DEFAULT_FRAGMENT_LENGTH + 1; i++) {
```

Comment:
Why +1? And please add a comment around the +1

Reply:
yes,  `+1`  is for testing a payload more than the allowed fragment size to see that s2n_record_write() correctly limits the fragment payload size to be the right length. this pattern is used throughout the other tests that is modified for this change.